Reference/google/protobuf/any.pb.swift

```diff
@@ -187,15 +187,7 @@ struct Google_Protobuf_Any: SwiftProtobuf.Message, SwiftProtobuf._MessageImpleme
   }
 
   func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V) throws {
-    try withExtendedLifetime(_storage) { (_storage: _StorageClass) in
-      try _storage.preTraverse()
-      if !_storage._typeURL.isEmpty {
-        try visitor.visitSingularStringField(value: _storage._typeURL, fieldNumber: 1)
-      }
-      if !_storage._value.isEmpty {
-        try visitor.visitSingularBytesField(value: _storage._value, fieldNumber: 2)
-      }
-    }
+    try _storage.traverse(visitor: &visitor)
     try unknownFields.traverse(visitor: &visitor)
```

Comment:
I believe it actually does end up correct, but I'll add some tests today to verify this.